src/nodes.coffee

```diff
@@ -1751,11 +1751,10 @@ exports.Class = class Class extends Base
       new Block expressions
 
   # Add an expression to the class initializer
-  #
-  # NOTE Currently, only methods and static methods are valid in ES class initializers.
-  # When additional expressions become valid, this method should be updated to handle them.
```

Comment:
Is this comment no longer valid?

Reply:
I wavered on it a little. I felt it was confusing since the check now also allows `PassthroughLiteral`s, and I didn't want to add that to the comment.

Reply:
I guess I could go the other way and flesh it out a little to indicate that it's the key method for determining whether something appears in the class initializer or executable body?

Reply:
Yeah I think more is better. Maybe even explicitly mention class fields/class properties, that when they reach Stage 4 this is where we detect them. That way future PRs to add things like that look more like your version of this PR and less like mine 😄